src/pools/D3MAavePool.sol

```diff
@@ -66,6 +66,47 @@ interface LendingPoolLike {
     );
 }
 
+// Need to use a struct as too many variables to return on the stack
+struct ReserveDataV3 {
```

Comment:
Same here, suggest to move struct definition to the interface.

Reply:
[done](https://github.com/makerdao/dss-direct-deposit/pull/93/commits/17c7a15b8706ed516f58f7de63e64e5a1d3701f8)